packages/connectivity/connectivity/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.5.0
+
+* No Location Permission using preprocessor flag.  Please check the [README](https://github.com/flutter/plugins/blob/master/packages/connectivity/connectivity/README.md#No-Location-Permission) for information on how to do that.
```

Comment:
It took them 45 years but Microsoft Word has finally spoken about 2 spaces after a period: https://www.cnn.com/2020/04/27/tech/microsoft-word-two-spaces-trnd/index.html

As a general principle, you should maintain the same convention that's there in the existing code.

Reply:
Good to know!  updated